keps/sig-node/3610-env-injection/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Env Injection
+kep-number: 3610
+authors:
+  - "@pacoxu"
+owning-sig: sig-node
+participating-sigs:
+- sig-api-machinery
+status: implementable
+creation-date: 2022-10-10
+reviewers:
+approvers:
+  - deads2k
+  - fedebongio
+  - lavalamp
+prr-approvers:
```

Comment:
Think this is no more a valid field in the `kep.yaml`

Reply:
xref: https://github.com/kubernetes/enhancements/pull/3569

Reply:
Thanks for your review. 

But according to https://github.com/kubernetes/enhancements/pull/3612#discussion_r1063176652, this feature is hard to go to GA like PodPreset. I'd like to close this PR.